README.md

```diff
@@ -74,7 +86,47 @@ Additionally, our releases contain additional information about what's up with u
 
 3. Start the PATH Container
 
-   Run `make path_up` to start the PATH container.
+   Run `make path_up_build_gateway` to start & build PATH or `make path_up` if this is not your first time.
```

Comment:
i dont know if it's typical or if it was just a "me" problem, but the `make` command requires `sudo` and it may be worthwhile to call out in this tutorial for total noobs.

Reply:
this was solved by https://github.com/jgsqware/clairctl/issues/60#issuecomment-358698788